ts/Series/TiledWebMap/TiledWebMapSeries.ts

```diff
@@ -0,0 +1,698 @@
+     * @param  {Highcharts.MapLonLatObject} lonLat
+     *         The map coordinates
+     * @return {Highcharts.PositionObject}
+     *         Array of x and y positions of the tile
+     */
+
+    public lonLatToTile(
+        lonLat: Highcharts.MapLonLatObject,
+        zoom: number
+    ): PositionObject {
+        const { lon, lat } = lonLat,
+            xTile = Math.floor(
+                (lon + 180) / 360 * Math.pow(2, zoom)
+            ),
+            yTile = Math.floor(
+                (
+                    1 - Math.log(
+                        Math.tan(lat * Math.PI / 180) +
+                        1 / Math.cos(lat * Math.PI / 180)
+                    ) / Math.PI
+                ) /
+                2 * Math.pow(2, zoom)
+            );
+        return { x: xTile, y: yTile };
+    }
+
+    /**
+     * Convert tile to map coordinates in longitude/latitude
+     *
+     * @function Highcharts.MapView#tileToLonLat
+     * @since  next
+     * @param  xTile
+     *         Position x of the tile
+     * @param  yTile
+     *         Position y of the tile
+     * @param  zTile
+     *         Zoom of the tile
+     * @return {Highcharts.MapLonLatObject}
+     *         The map coordinates
+     */
+
+    public tileToLonLat(
+        xTile: number,
+        yTile: number,
+        zTile: number
+    ): Highcharts.MapLonLatObject {
+        const lon = xTile / Math.pow(2, zTile) * 360 - 180,
+            n = Math.PI - 2 * Math.PI * yTile / Math.pow(2, zTile),
+            lat = (
+                180 /
+                Math.PI * Math.atan(0.5 * (Math.exp(n) - Math.exp(-n)))
+            );
+        return { lon, lat };
+    }
+
+    public drawPoints(): void {
+        const {
+                chart
+            } = this,
+            mapView = chart.mapView;
+
+        if (!this.tiles) {
+            this.tiles = {};
+        }
+        if (!this.transformGroups) {
+            this.transformGroups = [];
+        }
+        if (!mapView) {
+            return;
+        }
+
+        const {
+                tiles,
+                transformGroups
+            } = this,
+            options = this.options,
+            provider = options.provider,
+            { zoom } = mapView,
+            worldSize = 400.979322,
+            tileSize = 256;
+        let zoomFloor = zoom < 0 ? 0 : Math.floor(zoom),
+            maxTile = Math.pow(2, zoomFloor),
+            scale = ((tileSize / worldSize) * Math.pow(2, zoom)) /
+                ((tileSize / worldSize) * Math.pow(2, zoomFloor)),
+            scaledTileSize = scale * 256;
+
+        if (provider && (provider.type || provider.url)) {
+            if (provider.type && !provider.url) {
+                const ProviderDefinition =
+                    TilesProvidersRegistry[provider.type];
+
+                if (!defined(ProviderDefinition)) {
+                    error(
+                        'Provider cannot be reached.',
+                        false
+                    );
```

Comment:
The error message suggests that a connection attempt was made but it looks like the situation is a bit different - could be a type in the config or someone forgot to add the provider to the registry. "Tile Provider '${provider.type}' not defined in the Provider Registry." seems to be a more truthful error message in my opinion.